release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -141,6 +141,13 @@ A xref:../networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.
 When configuring an egress firewall rule, you can now use a xref:../networking/ovn_kubernetes_network_provider/configuring-egress-firewall-ovn.adoc#domain-name-server-resolution_configuring-egress-firewall-ovn[DNS domain name] instead of an IP address.
 With the addition of DNS support in the OVN-Kubernetes cluster network provider egress firewall implementation, parity is achieved with the OpenShift SDN cluster network provider egress firewall implementation.
 
+[id="ocp-4-7-preparation-HAProxy-2.2"]
+==== Adjusting HTTP header names
+
+If legacy applications are sensitive to the capitalization of HTTP header names, use the Ingress Controller `spec.httpHeaders.headerNameCaseAdjustments` API field for a solution to accommodate legacy applications until they can be fixed.
```

Comment:
I was going to suggest an example. Not sure if that's the right place for that in a RN.

Reply:
That's a good idea, thanks! I'll check to see if we have an example documented anywhere to link. If not, I'll open a second PR.

Reply:
I'll will open a separate PR to add an example and update the parameters table.